bin/create-bioentities-suggesters.sh

```diff
@@ -12,6 +12,11 @@ COLLECTION=${SOLR_COLLECTION:-"bioentities-v${SCHEMA_VERSION}"}
 
 #############################################################################################
 
+printf "\n\nDelete search component for suggesters...\n"
+curl $SOLR_AUTH -X POST -H 'Content-type:application/json' --data-binary '{
```

Comment:
quoting variables:
```suggestion
curl "$SOLR_AUTH" -X POST -H 'Content-type:application/json' --data-binary '{
```

Reply:
Addressed in 9c698367950458b6218006ae673de0b01ed51a0b.